cdk/lib/cdk-stack.ts

```diff
@@ -52,16 +52,12 @@ export class CdkStack extends GuStack {
       description: "Lambda to notify about instances with missing tags",
       timeout: Duration.seconds(30),
       memorySize: 512,
-      vpc: GuVpc.fromId(this, "vpc", parameters.vpc.valueAsString),
+      vpc: GuVpc.fromId(this, "vpc", { vpcId: parameters.vpc.valueAsString }),
```

Comment:
This changed [here](https://github.com/guardian/cdk/pull/149/files#diff-167082b85b20aeff983069d8655e08f9fa824bc77db741afd8fed1afcbb536a2R17).

Reply:
There's also now the `GuVpc.fromIdParameter` and `GuVpc.subnetsFromParameter` methods which would save defining those parameters explicitly. It looks up the values from parameter store so they may need adding there first.

Reply:
Cool, I'll set this up now.